### DISCUSSION TITLE
Reboot, Restart, Reset, Shutdown

### DISCUSSION THREAD

**Discussion by imanuk** [gopro/OpenGoPro#407] [Ideas]
In the project we have using USB connected GoPros with the API, we have a problem that prompts the need for a reboot.

Reboot's are not the fix and we are trying to understand the nature of the issue to avoid the reboot need from occurring.

However in the mean time we are frequently having to reboot the camera manually. 

Labs QR !OR reboots the camera and resolves our issue. It would be amazing to have a command we can send that reboots the camera in the same way.

**Comment by NickPronto** [comment]
Why not just suspend the keep alive command, and then wake again with BLE?

**Comment by tony-gutierrez** [comment]
It would be amazing to be able to do all kinds of labs things via http.... Have you tried using the labs WAKE command and cycling power to reboot?
